Demo/Sources/SwiftUI/HTMLTextDemoViewController.swift

```diff
@@ -50,17 +51,34 @@ extension HTMLTextDemoViewController: TweakableDemo {
     }
 
     func configure(forTweakAt index: Int) {
+        let spanMapper: HTMLStringSwiftUIStyleTranslator.SpanMapper = { attributes, currentStyle in
+            for attribute in attributes {
+                guard attribute.name == "style",
+                      attribute.value == "color:tjt-price-highlight" else {
```

Comment:
would be nice not to hardcode this here and in HTMLText as well

Reply:
For the demo I think it's OK, since the purpose is to test the span mapping. I could call it differently, I just didn't want to change the test HTML strings. 🙂